packages/shared/src/styles/constants.scss

```diff
@@ -133,6 +133,7 @@ $COLOR_LIGHT_GRAY_2: rgba(153, 156, 172, 0.32);
 $COLOR_LIGHT_GRAY_3: #eaeced;
 $COLOR_LIGHT_GRAY_4: #6e6e6e;
 $COLOR_LIGHT_GRAY_5: #c2c2c2;
+$COLOR_LIGHT_GRAY_6: #f2f3f4;
```

Comment:
This is already defined as `$color-grey-2`

Reply:
done ✅